database/ticket_test.go

```diff
@@ -332,3 +362,110 @@ func testCountTickets(t *testing.T) {
 
 	count("revoked", 1, 1, 1)
 }
+
+func tAltHist() *AltSigHistory {
+	return &AltSigHistory{
+		Addr: randString(35, addrCharset),
+		Req:  randBytes(1000),
+		Sig:  randBytes(72),
+	}
+}
+
+func testAddAltSigHistory(t *testing.T) {
+	ticket := exampleTicket()
+
+	// This should error if no ticket exists for hash in the database.
+	if err := db.AddAltSigHistory(ticket.Hash, tAltHist()); err == nil {
+		t.Fatal("expected error for no ticket in db")
+	}
+
+	// Insert a ticket into the database.
+	err := db.InsertNewTicket(ticket)
+	if err != nil {
+		t.Fatalf("error storing ticket in database: %v", err)
```

Comment:
s / error storing ticket in database / unexpected error storing ticket in database.

Reply:
ok